extensions/wikia/Insights/specials/UnconvertedInfoboxesPage.class.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+class UnconvertedInfoboxesPage extends PageQueryPage {
+	const LIMIT = 1000;
+
+	function __construct( $name = 'UnconvertedInfoboxes' ) {
+		parent::__construct( $name );
+	}
+
+	public function isListed() {
+		return false;
+	}
+
+	public function sortDescending() {
+		return true;
+	}
+
+	public function isExpensive() {
+		return true;
+	}
+
+	public function doQuery( $offset = false, $limit = self::LIMIT ) {
+		return $this->fetchFromCache( $limit, $offset );
+	}
+
+	public function recache( $limit = false, $ignoreErrors = true ) {
+		$dbw = wfGetDB( DB_MASTER );
+
+		/**
+		 * 1. Get the new data first
+		 */
+		$nonportableTemplates = $this->reallyDoQuery();
+		$dbw->begin();
+
+		/**
+		 * 2. Delete the existing records
```

Comment:
It could be more specific.
What do you think about 
'Delete all existing records for current queryPage type'
do I guess that right?